pkg/cmd/controller/config.go

```diff
@@ -35,9 +37,11 @@ const (
 	infraServiceAccountControllerServiceAccountName             = "serviceaccount-controller"
 	iInfraServiceAccountPullSecretsControllerServiceAccountName = "serviceaccount-pull-secrets-controller"
 	infraBuildControllerServiceAccountName                      = "build-controller"
+	infraBuilderRoleBindingsControllerServiceAccountName        = "builder-rolebindings-controller"
 	infraBuildConfigChangeControllerServiceAccountName          = "build-config-change-controller"
 	infraDeploymentConfigControllerServiceAccountName           = "deploymentconfig-controller"
 	infraDeployerControllerServiceAccountName                   = "deployer-controller"
+	infraDeployerRoleBindingsControllerServiceAccountName       = "deployer-rolebindings-controller"
```

Comment:
Ditto - don't add things here as the service account for the rolebindings controller is created by another component.